### DISCUSSION TITLE
how do you get into the slack discussions?

### DISCUSSION THREAD

**Discussion by avnav0** [dagster-io/dagster#11487] [General]
it seems like you need an @elemtnl email.

is it limited to paying customers or something?  I have a slack account, and  am not able to see the discussions.

**Comment by rexledesma** [comment]
[dagster.io/slack](https://dagster.io/slack) should be working as intended now.

**Reply by avnav0** [reply]
don't know about that mate, im getting 404.  to be clear i'm just a new user of the awesome framework, with no affiliation or anything...
and if i go through the shared linke on the github, it shows something is wrong...

**Reply by rexledesma** [reply]
Try the link again? The link is https://dagster.io/slack.

**Reply by avnav0** [reply]
awesome!!

**Comment by jcauserjt** [comment]
The link is not working for me either :( https://dagster.io/slack

```
This link is no longer active
To join this workspace, you’ll need to ask the person who originally invited you for a new link.
```
